jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -212,16 +212,14 @@ def get_regions(self, region_type=None, list_of_subset_labels=None,
             the native data unit. If True, subsets are returned in the spectral
             axis display unit set in the Unit Conversion plugin.
 
-        return_sky_region : bool or None, optional
-            If None (default) or True, then the returned region will be ``SkyRegion`` if
-            the data is aligned by WCS, if ``wrt_data`` is set, or if the configuration
-            is Cubeviz and the data has a WCS'. If set to False, a ``PixelRegion`` object will
-            be returned.
-        wrt_data : str or None
-            Name of data to use for applying WCS to subset when returning as
-            a ``PixelRegion`` object. The default behavior is to use the WCS of the parent
-            of the subset, so this keyword overrides that to use the WCS of the data with
-            this name.
+        wrt_data : bool, str,  or None
+            If aligned by WCS and ``wrt_data`` is not False, return ``SkyRegion`` object. In this
+            case, if ``wrt_data`` is set to the name of data, return ``SkyRegion`` using the wcs of
+            that data. Otherwise, the default is to use the parent data of the subset. If the
+            parent of the subset is the Default orientation layer, the first data loaded into
+            ``data_collection`` will be used instead. Also, if aligned by pixels and ``wrt_data``
+            is True or set to the name of valid data, return ``SkyRegion``. Finally, if in the
+            cubeviz configuration, return ``SkyRegion``. Otherwise, return PixelRegion.
```

Comment:
this description seems to only apply to spatial subsets.  I'm assuming spectral subsets is a follow-up effort, but maybe we should at least state that in the description and raise errors for the spectral case?